src/languages/csharp.js

```diff
@@ -131,13 +134,17 @@ export default function(hljs) {
     'join',
     'let',
     'nameof',
+    'nint',
```

Comment:
Are some of these types?  We have a separate list of types and if so we should add types to that list.